src/it/scala/io/iohk/ethereum/txExecTest/util/DumpChainApp.scala

```diff
@@ -32,10 +32,17 @@ import io.iohk.ethereum.security.SecureRandomBuilder
 import io.iohk.ethereum.utils.{Config, NodeStatus, ServerStatus}
 import monix.reactive.Observable
 import org.bouncycastle.util.encoders.Hex
+import org.scalamock.clazz.Mock
```

Comment:
With `MockFactory` you shouldn't need this

Reply:
indeed